core/src/main/groovy/noe/server/Tomcat.groovy

```diff
@@ -655,6 +655,7 @@ class Tomcat extends ServerAbstract implements WorkerServer {
       enableJavaSsl = '<Connector port="' + this.mainHttpsPort.toString() + '" protocol="HTTP/1.1"' + nl +
               '          SSLEnabled="true" maxThreads="150" scheme="https" secure="true"' + nl +
               '          clientAuth="false" sslEnabledProtocols="TLSv1.1+TLSv1.2"' + nl +
+              '          keystoreFile="' + this.keystorePath + '"' + nl +
```

Comment:
@sgala, was this missed on purpose or just a bug?

Reply:
I don't see what do you mean, so probably it is a bug :)